prov/usnic/src/fi_ext_usnic.h

```diff
@@ -42,7 +42,7 @@
 #include <stdint.h>
 #include <net/if.h>
 
-#define FI_PROTO_RUDP 100
+#define FI_PROTO_RUDP (100 | FI_PROV_SPECIFIC)
```

Comment:
I think the way it is right now is okay, but I'm all for changing it to something that we're more certain of. So you're suggesting that `FI_PROV_SPECIFIC` be changed to `(1u << 31)` and `FI_PROTO_RUDP` be changed to `(100u | FI_PROV_SPECIFIC)`? 

We could always make them `(UINT32_C(1) << 31)` as well.